Generic's Generic planet lobby BG

SomewhatSad14 opened the pull request:
# Description

A new lobby background with planets in the shape of the big/little dipper.

# Changelog

:cl: Generic
- add: Added planets lobby screen was added
- tweak: Froze earth

SomewhatSad14 committed:
Hope this is not broken

DEATHB4DEFEAT commented on Resources/Prototypes/lobbyscreens.yml:
```diff
@@ -9,3 +9,7 @@
 - type: lobbyBackground
   id: Room
   background: /Textures/LobbyScreens/room.png
+  
+- type: lobbyBackground
+  id: Planets
+  background: /Textures/LobbyScreens/planets.png
```
I've not messed with lobby screens, just adding this prototype works?

SomewhatSad14 replied:
It worked on my development build so it probably will work on the release build

Pspritechologist replied:
Yep, that's all it is

DEATHB4DEFEAT commented on Resources/Textures/LobbyScreens/attributions.yml:
```diff
@@ -12,3 +12,8 @@
+
+- files: ["planets.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Generic#1557 on discord"
```
```suggestion
  copyright: "Generic#1557 (502973272435195933) on discord"
```

Pspritechologist commented on Resources/Prototypes/lobbyscreens.yml:
```diff
@@ -9,3 +9,7 @@
+  
+- type: lobbyBackground
+  id: Planets
+  background: /Textures/LobbyScreens/planets.png
```
Let's give it a slightly less generic name. I can foresee other "planet" lobby images being added in the future :P

SomewhatSad14 replied:
Changed id: planets to id: Dipper

Pspritechologist reviewed (changes requested):
What I mentioned about the file name. Other than that, looks good to me.

Pspritechologist commented:
You should edit the PR name and description to uh
Be something :P

SomewhatSad14 committed:
YML Updated

Changed Planets to - Dipper, because the planets make the Ursa major? constellation (might be Ursa minor)

SomewhatSad14 committed:
Apply suggestions from code review

Co-authored-by: DEATHB4DEFEAT <77995199+DEATHB4DEFEAT@users.noreply.github.com>

Pspritechologist commented on Resources/Textures/LobbyScreens/attributions.yml:
```diff
@@ -12,3 +12,8 @@
   license: "CC-BY-NC-SA-3.0"
   copyright: "Abyssal#4842 on discord / @aby_doodles on Twitter"
   source: "https://twitter.com/aby_doodles"
+
+- files: ["planets.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Generic#1557 (502973272435195933) on discord"
+  source: "https://github.com/Park-Station/Parkstation"
```
```suggestion

- files: ["planets.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "Generic#1557 (502973272435195933) on discord"
  source: "https://github.com/Park-Station/Parkstation"
  
```

Pspritechologist commented:
Edit the PR description to be something basic, it gets saved in the commit history.

SomewhatSad14 committed:
Update Resources/Textures/LobbyScreens/attributions.yml

Co-authored-by: Pspritechologist <81725545+Pspritechologist@users.noreply.github.com>

Pspritechologist commented:
Don't merge this until the pr description is like, *anything*. A changelog isn't needed, a checklist isn't needed, but just *state what it does*, please.

DEATHB4DEFEAT commented:
> Don't merge this until the pr description is like, _anything_. A changelog isn't needed, a checklist isn't needed, but just _state what it does_, please.

you have permission to edit it

DEATHB4DEFEAT commented:
![image](https://github.com/Park-Station/Parkstation/assets/77995199/1e9e29aa-e724-48fd-a4e2-9a9385bc5c4d)

Pspritechologist commented:
> > Don't merge this until the pr description is like, _anything_. A changelog isn't needed, a checklist isn't needed, but just _state what it does_, please.
> 
> you have permission to edit it

I am well aware.

SomewhatSad14 committed:
Merge branch 'master' into Planets.Lobby

SomewhatSad14 committed:
Update lobbyscreens.yml

DEATHB4DEFEAT commented on Resources/Textures/LobbyScreens/attributions.yml:
```diff
@@ -3,3 +3,18 @@
+  copyright: "Solbusaur#4343 (220208697429852160) on discord"
+  source: "https://github.com/space-wizards/space-station-14"
+
+- files: ["ssxiv.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Abyssal#4842 on discord / @aby_doodles on Twitter"
+  source: "https://twitter.com/aby_doodles"
+  
```
```suggestion
```

DEATHB4DEFEAT commented on Resources/Textures/LobbyScreens/attributions.yml:
```diff
@@ -3,3 +3,18 @@
+- files: ["planets.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Generic#1557 (502973272435195933) on discord"
+  source: "https://github.com/Park-Station/Parkstation"
```
```suggestion
  source: "https://github.com/Simple-Station/Parkstation"

```

SomewhatSad14 committed:
Update Resources/Textures/LobbyScreens/attributions.yml

Co-authored-by: DEATHB4DEFEAT <77995199+DEATHB4DEFEAT@users.noreply.github.com>

SomewhatSad14 committed:
Update Resources/Textures/LobbyScreens/attributions.yml

Co-authored-by: DEATHB4DEFEAT <77995199+DEATHB4DEFEAT@users.noreply.github.com>
